corehq/toggles/__init__.py

```diff
@@ -2169,3 +2169,14 @@ def _commtrackify(domain_name, toggle_is_enabled):
     Manager only if the question's inputted answer is different from the current value in the case.
     """
 )
+
+GOOGLE_SHEETS_INTEGRATION = StaticToggle(
+    'google-sheet-integration',
+    'Unlock the Google Sheets view in Exports',
+    TAG_CUSTOM,
```

Comment:
What made you choose `TAG_CUSTOM`? Its description includes "This feature flag was created for one specific project." which might be appropriate, but I don't know exactly how we plan to use this toggle. Figured it's worth mentioning `TAG_SAAS_CONDITIONAL` or `TAG_INTERNAL` might be more fitting.

Reply:
Sure, I just used `TAG_CUSTOM` because I saw it being used on the other toggles

Reply:
Offlined and realized that saas conditional makes the most sense in this scenario.